_data/team/roles/releases_and_deployment.yaml

```
@@ -0,0 +1,15 @@
+role: Releases and deployment
+tasks: 
+  - Monitoring upstream and downstream package changes
+  - Updates and fixes for upstream and downstream packages
+  - Managing future release plans and timing
+  - Carrying out releases
+  - Maintaining continuous deployment infrastructure
+  - Monitoring issues in core library and sub-projects
+current_leads:
+  - Irfan Alibay
+current_members:
+  - Fiona Naughton
+historical_members:
+  - Irfan Alibay
```

If we're saying these are the 2020-2023 leads, then Richard should be here too since he was the person in charge of releases prior to 1.1.0.

---

I would err on the side of inclusivity and add Richard
```suggestion
  - Irfan Alibay
  - Richard Gowers
```